src/System.IO.IsolatedStorage/src/System/IO/IsolatedStorage/IsolatedStorageFile.cs

```diff
@@ -7,6 +7,7 @@
 using System.Collections.Generic;
 using System.Security;
 using System.Runtime.InteropServices;
+using System.Diagnostics;
 using System.Diagnostics.Contracts;
```

Comment:
In my personal opinion, they are not very useful, however, its merits are still under debate (#503 , dotnet/corert#320, dotnet/roslyn#98) and I do see other libraries relying on it. So for now, I don't see any harm in letting them stay.